public/400.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Your request could not be handled (400)</title>
+  <style type="text/css">
+    body { background-color: #fff; color: #666; text-align: center; font-family: arial, sans-serif; }
+    div.dialog {
+      width: 25em;
+      padding: 0 4em;
+      margin: 4em auto 0 auto;
+      border: 1px solid #ccc;
+      border-right-color: #999;
+      border-bottom-color: #999;
+    }
+    h1 { font-size: 100%; color: #f00; line-height: 1.5em; }
+  </style>
+</head>
+
+<body>
+  <!-- This file lives in public/400.html -->
+  <div class="dialog">
+    <h1>Your request could not be handled.</h1>
+    <p>Please check the url and post data for syntax errors.</p>
+    <p>For item requests, ensure that a valid DRUID is specified.</p>
+  </div>
+</body>
+</html>
```

Comment:
This was included in case the app needs to use one.  At present, it uses flash messages instead.  No harm in leaving it in, along with other public pages that might not be used anyway.